### ISSUE TITLE
dice and loss

### ISSUE THREAD

**Issue by yishouheng** [xianlin7/SAMUS#10]
May I ask why I started with a high validation set and why the test DICE was very low?
![74176832af46645a4b0899ebe8fc098b](https://github.com/xianlin7/SAMUS/assets/143176271/1ad84598-2cdb-4935-9c26-00224ce44797)

**Comment by yishouheng** [comment]
I'm stupid, bother, there are a few classes, id is the number of classes, not the background, for example, a tumor is 1, id=1

**Comment by xianlin7** [comment]
> I'm stupid, bother, there are a few classes, id is the number of classes, not the background, for example, a tumor is 1, id=1

Yes, ID is the label value of each category in the corresponding dataset.
We have provided a more detailed explanation of ID in [SAMCT](https://github.com/xianlin7/SAMCT)
